src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewServiceTest.java

```diff
@@ -323,6 +324,8 @@ private void checkDocumentIsNotCreatedAndReturnsError(String expected) {
     @EnumSource(value = AdjournCaseTypeOfHearing.class)
     void willSetPreviewFileWithNullReasons_WhenReasonsListIsEmpty(AdjournCaseTypeOfHearing nextHearingType) {
         when(userDetailsService.buildLoggedInUserName(USER_AUTHORISATION)).thenReturn(JUDGE_FULL_NAME);
+        when(userDetailsService.buildLoggedInUserSurname(USER_AUTHORISATION)).thenReturn(JUDGE_LAST_NAME);
```

Comment:
it's interesting that you've had to update so many mocks but not that many assertions